x-pack/plugins/cases/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "arrow-body-style": ["error", "as-needed"]
+  }
+}
```

Comment:
Please put overrides like this in the root `.eslintrc` file

Reply:
ah neat didn't have an override yet for cases, I'll follow along there. thanks